lib/rubocop/cop/style/array_intersect.rb

```diff
@@ -10,6 +10,7 @@ module Style
       # * `(array1 & array2).any?`
       # * `(array1.intersection(array2)).any?`
       # * `array1.any? { |elem| array2.member?(elem) }`
+      # * `(array1 & array2).count > 0`
```

Comment:
I didn't add `size` and `length` here, for brevity. Should I include them as well?

Reply:
At least one of them, as `size` and `length` are aliases, but `count` is a different method altogether.

Reply:
(and without a block, normally you shouldn't use `count` IMO)